### PR TITLE
Slot Resv endpoint update 1

### PR DESCRIPTION
**Pull request by Arnoldosmium**
Tried my best to figure out the current signatures.

### CODE REVIEW

File: signatures/reservations.yml [hunk 1]
```diff
@@ -5,22 +5,25 @@ Reservation:
...
         type: UUID
         description: Mandatory reservation slot ID.
-      user_id:
+      customer:
         type: UUID
         description: Mandatory customer ID.
-    returns: {}
+    returns:
+      rid:
+        type: String
```
Comment (fgao22) [inline signatures/reservations.yml]: This seem to be a UUID https://gitlab.com/yiconfirmed/backend/blob/develop/reservation/models.py#L16

File: signatures/slot.yml [hunk 2]
```diff
@@ -115,3 +118,26 @@ Slot:
         type: Object[]
         description: Successfully reset slot
     errors: {}
+  AdminSlotReset:
+    path: /slot/admin/reset
+    method: POST
+    title: Admin reset Slot availablity
+    description: reset a slot
+    params:
+      hospital:
```
Comment (fgao22) [inline signatures/slot.yml]: Do you mind adding "hospital_name" and "disease_name", as String fields in the response? We are about to add these two fields to save some API calls for the frontend.

File: signatures/slot.yml [hunk 3]
```diff
@@ -115,3 +118,26 @@ Slot:
...
+        type: UUID
+        description: hospital_id
+      disease:
+        type: UUID
+        description: disease_id
+      date:
+        type: Optional<Date>
+        description: date of time_slot to reset
```
Comment (fgao22) [inline signatures/slot.yml]: Could you add an example of the `date` field?

Comment (gabrielwry) [reply signatures/slot.yml]: Actually, we will soon remove this field once the cron job is up.

**Pull request merged**